lib/pundit.rb

```diff
@@ -56,11 +56,19 @@ def policy!(user, record)
     end
   end
 
+  module Helper
+    def policy_scope(scope)
+      policy_scopes[scope] ||= Pundit.policy_scope!(pundit_user, scope)
+    end
+  end
+
   included do
+    helper Helper if respond_to?(:helper)
     if respond_to?(:helper_method)
-      helper_method :policy_scope
       helper_method :policy
       helper_method :pundit_user
+      helper_method :policies
```

Comment:
Why is this method added as a `helper_method` when there doesn't seem to be any place where it is used? I see you're using the `policy_scopes` in the `Helper#policy_scope` method so I guess thats why  it needs to be declared as a helper method below.